.travis.yml

```diff
@@ -0,0 +1,32 @@
+language: c
+sudo: required
```

Comment:
AFAIK, using `sudo` on Travis locks you into what they call "legacy" infrastructure, which is slow and has high latency. See https://docs.travis-ci.com/user/migrating-from-legacy/.

We don't seem to need sudo here, because all packages can be installed using `apt` addon. See the travis.yml file in the arduino core for an example: https://github.com/esp8266/Arduino/blob/master/.travis.yml#L6-L11 and the official docs: https://docs.travis-ci.com/user/installing-dependencies/#Installing-Packages-on-Container-Based-Infrastructure.